application/controllers/SuggestController.php

```diff
@@ -58,4 +62,59 @@ public function timezoneAction(): void
 
         $this->getDocument()->addHtml($suggestions->forRequest($this->getServerRequest()));
     }
+
+    public function recipientAction(): void
+    {
+        $suggestions = new SearchSuggestions(
+            (function () use (&$suggestions) {
+                $createExcludeFilter = function (string $for, array $from): Filter\None {
+                    $toExclude = array_filter(array_map(function ($term) use ($for) {
+                        if (str_contains($term, ':') === false) {
+                            return '';
+                        }
+
+                        [$type, $id] = explode(':', $term, 2);
+
+                        return $type === $for ? $id : '';
+                    }, $from));
+
+                    $filter = Filter::none();
+                    if (! empty($toExclude)) {
+                        $filter->add(Filter::equal('id', $toExclude));
+                    }
+
+                    return $filter;
+                };
+
+                $contactFilter = Filter::all(
+                    $createExcludeFilter('contact', $suggestions->getExcludeTerms()),
+                    Filter::like('full_name', $suggestions->getSearchTerm())
+                );
+                foreach (Contact::on(Database::get())->filter($contactFilter) as $contact) {
+                    yield [
+                        'search' => 'contact:' . $contact->id,
+                        'label' => $contact->full_name,
+                        'type' => $this->translate('Contacts')
```

Comment:
Please provide the group label in the callback. Derive it from the search value with str_starts_with.